.changeset/thirty-cameras-explain.md

```diff
@@ -0,0 +1,13 @@
+---
+"@logto/core": minor
+---
+
+Update token grant to support organization API resources.
+
+Organization roles can be assigned with scopes (permissions) from the API resources, and the token grant now supports this.
+
+Once the user is consent to an application with "resources" assigned, the token grant will now include the scopes inherited from the organization roles.
+
+Users can narrow down the scopes by passing `organization_id` when granting an access token, and the token will only include the scopes from the organization roles of the specified organization, the access token will contain an extra claim `organization_id` to indicate the organization the token is granted for. Then the resource server can use this claim to protect the resource with additional organization-level authorization.
```

Comment:
doubting the usage of "narrow down", since the resource scopes can actually be wider with an `organization_id`